test/helper/sending.bash

```diff
@@ -45,57 +47,127 @@ function _send_email() {
       ( '--server' ) SERVER=${2:?--server given but no argument} ; shift 2 ;;
       ( '--port' )   PORT=${2:?--port given but no argument}     ; shift 2 ;;
       ( '--data' )
-        local TEMPLATE_FILE="/tmp/docker-mailserver-test/emails/${2:?--data given but no argument provided}.txt"
-        FINAL_SWAKS_OPTIONS+=('--data')
-        FINAL_SWAKS_OPTIONS+=('-')
-        FINAL_SWAKS_OPTIONS+=('<')
-        FINAL_SWAKS_OPTIONS+=("${TEMPLATE_FILE}")
+        ADDITIONAL_SWAKS_OPTIONS+=('--data')
+        local FILE_PATH="/tmp/docker-mailserver-test/emails/${2:?--data given but no argument provided}"
+        if _exec_in_container_bash "[[ -e ${FILE_PATH} ]]"; then
+          ADDITIONAL_SWAKS_OPTIONS+=("@${FILE_PATH}")
+        else
+          ADDITIONAL_SWAKS_OPTIONS+=("'${2}'")
```

Comment:
This would be a regression, because this way, you could not provide "inline" data, i.e. something like `--data 'Date: 1 Jan 2024\nSubject: Hello Mom!'`. I can solve this though and have the function apply a default.